build.gradle

```diff
@@ -1,6 +1,7 @@
 apply plugin: 'java'
 apply plugin: 'maven'
 apply plugin: 'signing'
+apply plugin: 'eclipse'
```

Comment:
Can this come out?  We try to avoid putting IDE-specific stuff in the build (and all use IntelliJ anyway!)